README.md

```diff
@@ -1,28 +1,19 @@
-# Open Model Manager Resources
-This repository contains the start-up script for SAS Open Model Manager, as well as helper scripts for administration and customization. In addition there are sample models and data, and examples of using Jupyter notebooks to submit API requests for common tasks. 
-
-For more information about deploying the container, see [SAS Open Model Manager 1.2 for Containers: Deployment Guide](http://documentation.sas.com/?docsetId=dplymdlmgmt0phy0dkr&docsetTarget=titlepage.htm&docsetVersion=1.2&locale=en).
+# Model Management Resources for Open-Source Models
+This repository contains resources that can be used for administration and customizations that help with managing open-source models. In addition, there are sample models and data, and examples of using Jupyter notebooks to submit API requests for common tasks for managing models using SAS Model Manager or SAS Open Model Manager.
```

Comment:
> [...] examples of using Jupyter notebooks to submit [...]

This wording is a little awkward. Can we change this to:

> [...] example Jupyter notebooks which can be used to submit [...]

?

Reply:
I am good with this change.